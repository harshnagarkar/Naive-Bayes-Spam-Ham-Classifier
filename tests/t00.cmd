 toy_trainHam.txt toy_trainSpam.txt toy_testHam.txt toy_testSpam.txt 1
