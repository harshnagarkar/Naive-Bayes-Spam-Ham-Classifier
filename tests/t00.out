ham -10.2041 -10.2373
spam -6.64758 -5.51289
ham -4.45036 -6.6115
spam -11.3027 -9.54418
ham -8.22312 -8.77099
spam -11.9959 -9.94964
0.666667 0.666667 0.666667
